runner/src/server/views/help/privacy.html

```diff
@@ -0,0 +1,15 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Privacy Notice
+{% endblock %}
+
+{% block content %}
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">How we use your data (Privacy Notice)</h1>
```

Comment:
We can't use the content NM has written for lists - it's very specific to FCDO and the lists (apply) service. This better than what's currently there, currently it just defaults to "#". With this mechanism the privacy template can either be replaced (because there's something to replace), or privacyPolicyUrl is used

Reply:
One last consideration: documentation.  I think the README needs to be updated to include the privacy notice environment variable.